containers/scripts/build_strongswan.sh

```diff
@@ -26,8 +26,13 @@ write_stronglahav_version() {
 config_strongswan() {
 	cd /strongswan
 
-	./autogen.sh
```

Comment:
Is `./autogen.sh` step removal intentional? was it moved somewhere else?
There is no `configure` script without prior `autoreconf`.

Reply:
Right, fixed